### PR TITLE
UI-Profile-Page

### PR DESCRIPTION
**Pull request by rolandlor**

### CODE REVIEW

**Review by ottpeter — APPROVED** [review]

File: src/MainRouter.tsx [hunk 1]
```diff
@@ -106,15 +110,21 @@ const MainRouter = (): ReactElement => {
...
       <Route path={ROUTES.ROOMS} element={<RoomsPage />} />
-      <Route path={"/chat_dev"} element={<Chat
-          topic={TOPIC}
-          privKey={PRIVKEY}
-          stamp={STAMP as BatchId}
-          nickname={username}
-          gsocResourceId={GSOC_RESOURCE_ID}
-      />} />
+      <Route
+        path={"/chat_dev"}
```
Comment (bosi95) [inline src/MainRouter.tsx:114]: add path to ROUTES enum in constants

File: src/pages/Profile/Profile.tsx [hunk 2]
```diff
@@ -1,90 +1,56 @@
...
-            fontWeight: "400",
-            textAlign: "left",
-            marginLeft: "17px",
-            marginBottom: "14px",
-          }}
-        >
-          Each active registration you got extra 2 points
-        </div>
+    <div className="profile">
+      <NavigationHeader to="/home" />
```
Comment (bosi95) [inline src/pages/Profile/Profile.tsx:14]: use path from ROUTES enum

File: src/pages/Profile/Profile.tsx [hunk 3]
```diff
@@ -1,90 +1,56 @@
...
+            showContent={true}
+          />
+          <ProfileBox
+            title="Get referal points"
+            link="Share referal link"
+            showPoints={false}
+            showContent={true}
+          />
+          <Link
+            to="/how-does-it-work"
```
Comment (bosi95) [inline src/pages/Profile/Profile.tsx:45]: use ROUTES

File: src/pages/HowDoesItWork/HowDoesItWork.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+import "./HowDoesItWork.scss";
+import NavigationHeader from "../../components/NavigationHeader/NavigationHeader";
+import HomeBackground from "../../assets/welcome-glass-effect.png";
+import CollectPointsImg from "../../assets/collect-points.png";
+import ProfileGetBZZImg from "../../assets/profile-get-BZZ.png";
+
+const HowDoesItWork: React.FC = () => {
+  return (
+    <div className="how-does-it-work-page">
+      <NavigationHeader to="/profile" />
```
Comment (bosi95) [inline src/pages/HowDoesItWork/HowDoesItWork.tsx:11]: use ROUTES

**Review by bosi95 — APPROVED** [review]
looks okay, just some minor comments

**Review by Kexort — APPROVED** [review]
Nice!

File: src/MainRouter.tsx (hunk 1 above)
Comment (rolandlor) [reply src/MainRouter.tsx:114]: It will be corrected it in another PR

File: src/pages/Profile/Profile.tsx (hunk 2 above)
Comment (rolandlor) [reply src/pages/Profile/Profile.tsx:14]: It will be corrected it in another PR

File: src/pages/Profile/Profile.tsx (hunk 3 above)
Comment (rolandlor) [reply src/pages/Profile/Profile.tsx:45]: It will be corrected it in another PR

File: src/pages/HowDoesItWork/HowDoesItWork.tsx (hunk 4 above)
Comment (rolandlor) [reply src/pages/HowDoesItWork/HowDoesItWork.tsx:11]: It will be corrected in another PR

**Pull request merged**